generator/template.go

```diff
@@ -581,6 +581,8 @@ func mapWellKnownType(protoType string) string {
 		return "StructPBValue[]"
 	case ".google.protobuf.Struct":
 		return "{ [key: string]: StructPBValue }"
+	case ".google.protobuf.FieldMask":
+		return "string | null"
```

Comment:
I see [here](https://googleapis.dev/nodejs/vision/latest/google.protobuf.IFieldMask.html) that `FieldMask` type is `Array.<string> \| null` so should this return `"string[] | null"`?